packages/circuit-ui/components/Modal/Modal.module.css

```diff
@@ -18,157 +41,154 @@
   );
 }
 
-@media (max-width: 479px) {
-  .base {
-    right: 0;
-    bottom: 0;
-    left: 0;
-    transition: transform var(--cui-transitions-default);
-    transform: translateY(100%);
-  }
+/* Close button */
+.base .close {
+  position: absolute;
+  z-index: var(--cui-z-index-absolute);
+}
 
-  .base::after {
-    height: var(--cui-spacings-mega);
-  }
+/* Native Backdrop */
+.base::backdrop {
+  background: transparent;
+}
+
+/* Polyfill Backdrop */
+.base + .backdrop {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: transparent;
```

Comment:
Given that the backdrop is now simulated using a box-shadow to make the animation work on Firefox (we should add a code comment for this hack), does the regular backdrop still need to be positioned and animated? Maybe to intercept outside clicks?

Reply:
good catch, i think its safe to remove the animations, but the positioning is used for the outside clicks.